views/openacademy_session_views.xml

```diff
@@ -31,7 +31,7 @@
         <field name="name">openacademy.session.view.tree</field>
         <field name="model">openacademy.session</field>
         <field name="arch" type="xml">
-            <tree>
+            <tree decoration-info="duration<5" decoration-danger="duration>15">
```

Comment:
does < and > work? Don't you need to escape them?

CC @ajqn9094

Reply:
I think it would not merit escape < and >. That is, it is assumed that in those attributes go comparisons, not xml/html text.

Reply:
I think that for these cases it is always preferable to use `&lt;` and `&gt;` instead of `<` and `>` for topics that can be taken as start and end of tags

Reply:
Okay, got it.